.github/workflows/gradle-check.yml

```diff
@@ -92,12 +99,7 @@ jobs:
           issue-number: ${{ env.pr_number }}
           body: |
             ### Gradle Check (Jenkins) Run Completed with:
-            * **RESULT:** ${{ env.result }} :grey_exclamation:
-            * **FLAKY TEST FAILURES:**
-            The following tests failed but succeeded upon retry:
```

Comment:
This is in the "Create Comment Flaky" step which is only executed when the overall result is a success, which _should_ mean that all these failures are flaky. Do you have a counter example where it was wrong?

Reply:
Oh I see what happened here. My bad. It's a lot of duplication to add this text given the newer code, let's talk about it in https://github.com/opensearch-project/OpenSearch/pull/5297?